CDCgov/NEDSS-Modernization
# CNFT2-1179-page-rule-api

## Description

Changing structure of Page Rule Create API endpoint and adding full test coverage using new structure of BDD testing using gerkin statements.

## Tickets

* [CNFT2-1179](https://cdc-nbs.atlassian.net/browse/CNFT2-1179)

## Steps to verify

1. When running on local, can use swagger api to manually create request and check the database for the addition of the specific rule.
2. Can run JUNIT and cucumber tests to check for proper implementation as well


[CNFT2-1179]: https://cdc-nbs.atlassian.net/browse/CNFT2-1179?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
